pkg/run/storage.go

```diff
@@ -79,7 +117,17 @@ func (r *RunStorageService) Write(ctx context.Context, bucket string, key string
 		return err
 	}
 
-	return r.StorageService.Write(ctx, bucket, key, object)
+	err = r.StorageService.Write(ctx, bucket, key, object)
+	if err != nil {
+		return err
+	}
+
+	err = r.triggerBucketNotification(ctx, bucket, key, v1.BucketNotificationType_Created)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
```

Comment:
The writer shouldn't care if the responders error.

this should be something like 
```go
go r.triggerBucketNotification(ctx, bucket, key, v1.BucketNotificationType_Created)
```

side log the errors in the triggerBucketNotification method instead